frontend/js/main.js

```diff
@@ -15,7 +15,8 @@ function submitScript(text) {
 }
 
 function checkGenre(text, callback) {
-    $.ajax("https://romcomnotromcom.azurewebsites.net/predict", {
+    console.log(`${API_BASE}/predict`)
```

Comment:
Probably want to remove this `console.log`

Reply:
Yeah, done